Meta/Azure/Serenity.yml

```diff
@@ -37,6 +37,7 @@ jobs:
         cmake -S Meta/CMake/Superbuild -B Build/superbuild -GNinja \
           -DSERENITY_ARCH=${{ parameters.arch }} \
           -DSERENITY_TOOLCHAIN=Clang \
+          -DENABLE_ALL_KERNEL_MODULES=ON \
```

Comment:
Is the intention to always boot in CI with all modules? or just make sure they compile?

If we only care about compilation then adding `set(ENABLE_ALL_KERNEL_MODULES ON)` to the top level CMakeLists.txt instead of to each CI job would be cleaner

(line 114 in latest master)
```
if (ENABLE_ALL_DEBUG_FACILITIES)
    set(ENABLE_ALL_THE_DEBUG_MACROS ON)
    set(ENABLE_EXTRA_KERNEL_DEBUG_SYMBOLS ON)
```

Reply:
The initial intention was indeed to ensure the code doesn't rot, but I also want to ensure we don't make it to rot in runtime, hence we try to make the best effort we can to ensure it boots OK with all modules being compiled into the Kernel.

Reply:
If you feel like I didn't explain it well enough, feel free to unresolve this conversation.